### ISSUE TITLE
.Net: VS2022 NuGet can't find Microsoft.SemanticKernel

### ISSUE THREAD

**Issue by JDRay42** [microsoft/semantic-kernel#3426]
I have a freshly-installed version of VisualStudio 2022 running on Windows. I want to get started with SemanticKernel, so I tried following the instructions for **Getting Started with C#** in the README.  Step two says, "Add the semantic kernel nuget `Microsoft.SemanticKernel`", which I tried using both the PM console and the GUI interface.  Browsing for anything with *SemanticKernel* in the title doesn't find anything by Microsoft.

I checked the NuGet source settings, and it says **https://api.nuget.org/v3/index.json**, which I believe to be correct.

**Comment by JDRay42** [comment]
Update: Per advice from someone, I added the folder from my local repo clone to NuGet as a package source (.../repos/semantic-kernel/dotnet/nuget) and tried to find it that way.  Still no luck.

**Comment by matthewbolanos** [comment]
I'm not sure why your system isn't able to find it... Here's the link to the nuget package: https://www.nuget.org/packages/Microsoft.SemanticKernel/. Are you able to use the command line instructions to get it added?

**Comment by JDRay42** [comment]
No, and I get this message when trying the command line:
`The 'Source' parameter is not respected for the transitive package management based project(s) SK-RAG. The enabled sources in your NuGet configuration will be used.` So I made sure the above path is in the Sources in NPM. Still no love.

I have tried everything I can think of. I've tried it with our VPN on, with it off, using a fully-qualified ID (microsoft.semantickernel)... I suspect the problem is with my installation rather than an overall bug if no one else is having the issue. Still, it's maddening.
